Source/Moq.Analyzers.Test/PackageTests.Baseline#manifest.verified.nuspec

```diff
@@ -0,0 +1,17 @@
+﻿<package>
+  <metadata>
+    <id>Moq.Analyzers</id>
+    <version>********</version>
+    <authors>Andrey "Litee" Lipatkin</authors>
+    <licenseUrl>https://github.com/Litee/moq.analyzers/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Litee/moq.analyzers</projectUrl>
+    <description>Roslyn analyzer that helps to write unit tests using Moq mocking library by highlighting typical errors and suggesting quick fixes. Port of Resharper extension to Roslyn. Find the full list of detected issues at project GitHub page.</description>
+    <releaseNotes>Upgraded to support Visual Studio 2019</releaseNotes>
+    <copyright>2015-2019 Andrey Lipatkin</copyright>
+    <tags>moq, mock, test, analyzers</tags>
+    <repository type="git" commit="****************************************" />
+    <dependencies>
+      <group targetFramework=".NETStandard2.0" />
```

Comment:
Since this is an analyzer only, there's no shipping code, and hence no dependencies, so I don't think this should be here. Tracked by #31.